docs/azure_arc_jumpstart/azure_arc_data/microk8s/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+type: docs
+title: "MicroK8s"
+linkTitle: "MicroK8s"
+weight: 4
```

Comment:
```suggestion
weight: 5
```

Since this comes before the "Upstream Kubernetes (Kubeadm)" section in the Jumpstart, also update this section with "_weight: 6_"

Reply:
Ah I thought this had to do with font weight, didn't realize [this](https://bwaycer.github.io/hugo_tutorial.hugo/content/ordering/) is how Hugo ordering works.

Updated microk8s, as well as kubeadm.